src/Backend/Core/Engine/Header.php

```diff
@@ -11,7 +11,7 @@
 
 use Symfony\Component\HttpKernel\KernelInterface;
 use MatthiasMullie\Minify;
-use Backend\Core\Engine\Language as BL;
+use Backend\Core\Language\Language as BL;
```

Comment:
Can you use BackendLanguage instead?
So it is consistent with your other changes.